### PR TITLE
new discord link

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: content/1_General/Contributing.mdx [hunk 1]
```diff
@@ -27,7 +27,7 @@ If you have any questions about how to contribute, feel free to just open an
 [issue](https://github.com/cpinitiative/usaco-guide/issues) or a
 [pull request](https://github.com/cpinitiative/usaco-guide/pulls) and a team
 member will be able to assist you. If you want, you can also
-[join our Discord server](https://discord.gg/UKbyewj2Ft).
+[join our Discord server](https://discord.gg/53zkXQH8).
 
 ![Discord](https://img.shields.io/discord/717477028072521797?label=Discord&style=for-the-badge)
```
Comment (maggie-j-liu) [inline content/1_General/Contributing.mdx:32]: this should also be updated, but when I try https://img.shields.io/discord/754884077542441002?label=Discord&style=for-the-badge it says `Widget disabled`, not sure how to fix?

Comment (SansPapyrus683) [reply content/1_General/Contributing.mdx:32]: maybe just remove it then?

Comment (thecodingwizard) [reply content/1_General/Contributing.mdx:32]: [__https://img.shields.io/discord/754884077542441002?label=Discord&style=for-the-badge](https://img.shields.io/discord/754884077542441002?label=Discord&style=for-the-badge)

Comment (thecodingwizard) [reply content/1_General/Contributing.mdx:32]: https://img.shields.io/discord/754884077542441002?label=Discord&style=for-the-badge

(I just enabled it)

Comment (SansPapyrus683) [reply content/1_General/Contributing.mdx]: alr i added it @thecodingwizard

**Review by maggie-j-liu — APPROVED** [review]

**Pull request merged**